src/Chapter1.hs

```diff
@@ -490,7 +490,8 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit = (`mod` 10) . (abs)
```

Comment:
Great implementation! We have not explained eta-reduction and function composition, but you got it right 💪🏼 

One minor thing: parenthesis are redundant here, you can clean it up a bit:

```suggestion
lastDigit = (`mod` 10) . abs
```

Reply:
Ack